gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.ts

```diff
@@ -98,22 +98,24 @@ export class CatalogComponent {
   }
 
   public onFilterSelection(event: string) {
+    this.filter = event
```

Comment:
When selecting a filter in the drop-down to the right, it is not purple + checkmark once selected.